configure/migrator/migrator.Job.yaml

```diff
@@ -0,0 +1,184 @@
+# These manifests are provided in the event that someone may wish to execute
+# database migrations outside of a code deployment.
+#
+# For more information, please see: https://docs.sourcegraph.com/admin/install/kubernetes/operations#database-migrations
+
+# ---
```

Comment:
You can uncomment this job - it won't get called automatically so it's safe to leave in the "active" state.

Reply:
Done!